backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -1020,7 +1047,7 @@ bool SurfaceSdlGraphicsManager::loadGFXMode() {
 	if (_tmpscreen2 == nullptr)
 		error("allocating _tmpscreen2 failed");
 
-	if (_videoMode.isHwPalette) {
```

Comment:
I think, in some cases, some video modes may have or not have a hardware palette. Now, it seems to me lock it on a platform level.

Reply:
That would be a nice thing to implement later, but at the moment no backends toggle hardware palettes based on what the current game requires. If we do implement this, it can be handled in the initGraphicsSurface function.